docs/resources/dataset.md

```diff
@@ -11,17 +11,20 @@ Creates a dataset.
 ```hcl
 resource "honeycombio_dataset" "my_dataset" {
   name = "My dataset"
+  description = "buzzing with data"
 }
 ```
 
 ## Argument Reference
 
 The following arguments are supported:
 
-* `name` - (Required) The name of the dataset.
+-   `name` - (Required) The name of the dataset.
```

Comment:
Not sure if this was a markdown linter, but in the name of consistency I'd like to stick to the `*` approach used everywhere else

Reply:
yep, this must of been a plugin of mine, was not intentional.